torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -1922,6 +1922,20 @@ def create_from_config(cls, config: CppMicroGemmConfig):
             alpha,
         )
 
+    def skip_amx_kernel_for_woq(m, block_m, dynamic_M, input_dtype, input2_dtype):
+        # For WoQ GEMM, AMX micro-kernel may not perform well if m is small.
+        # Exception: for dynamic shapes, we consider using the AMX micro-kernel.
+        if (
+            dynamic_M
+            or input_dtype != torch.bfloat16
+            or input2_dtype not in [torch.int8, torch.uint8]
+        ):
+            return False
+        # For WOQ INT8, use AMX for m >= block_m
+        # For WOQ INT4, use AMX for m >= 5
+        m_threashold = block_m if input2_dtype == torch.int8 else 5
```

Comment:
small nit: `m_threshold`. 

Not necessary.

Reply:
Thanks. It's fixed now.